### PR TITLE
Update SymfonyRequirements.php to better handle infinite values

### PR DESCRIPTION
**Pull request by gnutix**
With `master`, I'm getting the following output :

```
Optional recommendations to improve your setup
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

 * "memory_limit" should be greater than "post_max_size".
   > Set "memory_limit" to be greater than "post_max_size".

 * "post_max_size" should be greater than "upload_max_filesize".
   > Set "post_max_size" to be greater than "upload_max_filesize".
```

with configuration : 

```
upload_max_filesize = 500M
memory_limit = -1
post_max_size = 0
```

This PR removes these warnings as they make no sense.

### CODE REVIEW

File: src/SymfonyRequirements.php [hunk 1]
```diff
@@ -368,14 +378,14 @@ class_exists('PDO'),
      *
      * @return int - Converted size
      */
-    private function convertShorthandSize($size)
+    private function convertShorthandSize($size, string $infiniteValue = '-1')
```
Comment (Tobion) [inline src/SymfonyRequirements.php]: the requirements checker ismust currently be compatible with php 5.3. the string type breaks this.

File: src/SymfonyRequirements.php [hunk 2]
```diff
@@ -328,15 +328,25 @@ function ($cfgValue) { return $cfgValue > 100; },
 
         $this->addPhpConfigRecommendation(
             'post_max_size',
-            $this->getPostMaxSize() < $this->getMemoryLimit(),
+            function (): bool {
```
Comment (Tobion) [inline src/SymfonyRequirements.php]: incompatible with php 5.3

File: src/SymfonyRequirements.php [hunk 3]
```diff
@@ -328,15 +328,25 @@ function ($cfgValue) { return $cfgValue > 100; },
...
+            },
             true,
             '"memory_limit" should be greater than "post_max_size".',
             'Set "<strong>memory_limit</strong>" to be greater than "<strong>post_max_size</strong>".'
         );
 
         $this->addPhpConfigRecommendation(
             'upload_max_filesize',
-            $this->getUploadMaxFilesize() < $this->getPostMaxSize(),
+            function (): bool {
```
Comment (Tobion) [inline src/SymfonyRequirements.php]: incompatible with php 5.3

**Comment by gnutix** [comment]
I can't see how we could test for this edge case... Do you?

**Comment by fabpot** [comment]
Thank you @gnutix.

**Comment by gnutix** [comment]
@fabpot I had completely forgotten this PR, and re-reading it, it seems I introduced a typo in line 348 : \INFO instead of \INF. Would you mind fixing it? My apologies. 🙏

**Comment by fabpot** [comment]
@gnutix Thanks for the head up, fixed it and released.
